backend/controllers/whereabouts/whereaboutsHomepage.js

```diff
@@ -41,21 +41,7 @@ const whereaboutsTasks = [
   },
 ]
 
-module.exports = oauthApi => async (req, res) => {
-  try {
-    const userRoles = await oauthApi.userRoles(res.locals)
```

Comment:
None of the tasks on whereabouts homepage are role dependant so a call to oAuth is not needed.